utils/credentials/credentials.go

```diff
@@ -67,38 +68,39 @@ func (a *userCredentials) RequireTransportSecurity() bool {
 }
 
 // loadFromFile loads the certificates from files.
```

Comment:
loads a certificate key pair into a tls certificate and a CA certificate into a x509 certificate.